src/components/common/EdgeAnim.tsx

```diff
@@ -33,6 +34,12 @@ interface Anim {
 }
 
 interface Props extends AnimateProps<ViewProps> {
+  /**
+   * disable animation
+   * anim => disable animation but still render a container view
+   * view => render the children with no container view
+   * */
+  disable?: 'anim' | 'view'
```

Comment:
Why is this an enumeration? I think it's clear through the API and component name that a boolean true here means just the lack of animation. 

In general I think it's clearer to have named value enums, though the rest of our app has a pattern of `disable` being a boolean flag. One less thing to hover would be nice and usage would have less code. 

If there's future plans to have more disable configuration types then a different prop name seems more appropriate